### PR TITLE
Bug 1188300 - No notification when a download starts

### PR DESCRIPTION
**Pull request by jedireza**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=663dd397943655eb89224894cae8efc8551a1b57

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9546aa9d6483e8e0029703a52fa5141da0b6d3f6

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a38022863b6f819392b5529b095f4c475a67f1f3

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8efb96f282cf7c0f1fbe4f6723a712eb6c7423b7

File: apps/system/test/unit/download_notification_test.js [hunk 1]
```diff
@@ -96,14 +96,25 @@ suite('system/DownloadNotification >', function() {
...
+
+    test('The download continues', function() {
+      notification.state = 'downloading';
+      assert.isFalse(NotificationScreen.addNotification.called);
+      download.state = 'downloading';
+      download.listeners[0]();
+      assertUpdatedNotification(download);
+
+      var calls = NotificationScreen.addNotification.getCalls();
+      assert.isTrue(calls[0].args[0].noNotify);
```
Comment (etiennesegonzac) [inline apps/system/test/unit/download_notification_test.js]: nit: let's use the sam way to access the call like `NotificationScreen.addNotification.args[0].noNotify`

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f8aab29d06293a1e1097555f7c423298d07f5213

**Pull request merged**